cs/test/core/TypeAliasTests.cs

```diff
@@ -301,13 +302,15 @@ public void AliasBlob()
         public void AliasBonded()
         {
             var from = new BondedAlias {lazy = new Lazy<Foo>(UnitTest.Random.Init<Foo>())};
+            Assert.IsTrue(Reflection.IsBonded(typeof(Lazy<Foo>)));
```

Comment:
It might be a little better if the two asserts you added got their own tests - `CustomIBondedImpl_Reflection_IsBonded`, or something like that. No big either way.